components/pedigree/resources/src/main/resources/pedigree/view/workspace.js

```diff
@@ -139,17 +140,17 @@ define([
             // set display:block
             svgText = svgText.replace(/(<svg[^<>]+style=")/g, "$1display:block; ");
```

Comment:
1) no velocity/java in 1.2 (we start using it for family-studies only)

2) it is used not only for saving, but e.g. for print preview, which is completely inside JS/pedigree and will stay this way for 1.3/family-studies